httpreq/httpreq.go

```diff
@@ -127,11 +164,21 @@ func (r *HttpReq) Call(v interface{}) (int, error) {
 	if err != nil {
 		return 0, err
 	}
-
 	if v != nil {
-		if err := json.Unmarshal(b, &v); err != nil {
+		err = DataTypeFactory{}.New(r.dataType).unMarshal(b, v)
+		if err != nil {
 			return resp.StatusCode, errors.New(string(b))
 		}
 	}
 	return resp.StatusCode, nil
 }
+
+func setDefaultTransport() {
```

Comment:
Maybe no where use this function.
If noone use this function, remove it.

Reply:
I has deleted it